spec/inputs/file_tail_spec.rb

```diff
@@ -424,17 +451,26 @@
         end
       end
 
-      it "should only have one set of files open" do
+      it "should only actually open files when content changes are detected" do
         subject.register
         expect(lsof_proc.call).to eq("")
+        # first run processes the file and records sincedb progress
         run_thread_proc.call
-        sleep 0.25
+        sleep 0.2
```

Comment:
relying on such a sleep is definitely error prone. can't a more specify wait be used instead?